packages/amplify_flutter/lib/amplify_flutter.dart

```diff
@@ -28,7 +31,6 @@ export 'src/config/api/api_config.dart';
 export 'src/config/auth/auth_config.dart';
 export 'src/config/config_map.dart';
 export 'src/config/storage/storage_config.dart';
-
 // Utilities
 export 'src/utils/equatable.dart';
 export 'src/utils/json.dart';
```

Comment:
I can't leave a comment on the line for some reason but this:

```dart
/// Top level singleton Amplify object.
final AmplifyClass Amplify = AmplifyClass.protected();
```

should become this:

```dart
/// Top level singleton Amplify object.
AmplifyClass get Amplify => AmplifyClass.instance;
```

Reply:
Also maybe a setter as well?

Reply:
Good call. Updated and added a setter.